toolkit/cli/commands/src/address_association_signatures.rs

```diff
@@ -0,0 +1,104 @@
+use crate::key_params::MainchainSigningKeyParam;
+use anyhow::Ok;
+use byte_string::ByteString;
+use clap::Parser;
+use pallet_address_associations::AddressAssociationSignedMessage;
+use parity_scale_codec::Encode;
+use serde::Serialize;
+use serde_json::json;
+use sidechain_domain::*;
+use std::str::FromStr;
+
+#[derive(Clone, Debug, Parser)]
+#[command(author, version, about, long_about = None)]
+pub struct AddressAssociationSignaturesCmd<
+	PartnerchainAddress: Clone + Sync + Send + FromStr + 'static,
+> {
+	/// Genesis UTXO of the target Partner Chain
+	#[arg(long)]
+	pub genesis_utxo: UtxoId,
+	/// Partner Chain address to be associated with the Cardano address
+	#[arg(long, value_parser=parse_pc_address::<PartnerchainAddress>)]
+	pub partnerchain_address: PartnerchainAddress,
+	/// Cardano ECDSA signing key. Its public key will be associated with partnerchain_address.
+	#[arg(long)]
+	pub signing_key: MainchainSigningKeyParam,
+}
+
+fn parse_pc_address<T: FromStr>(s: &str) -> Result<T, String> {
+	T::from_str(s).map_err(|_| "Failed to parse Partner Chain address".to_owned())
+}
+
+impl<PartnerchainAddress> AddressAssociationSignaturesCmd<PartnerchainAddress>
+where
+	PartnerchainAddress: Serialize + Clone + Sync + Send + FromStr + Encode + 'static,
+{
+	pub fn execute(&self) -> anyhow::Result<()> {
+		let signature = self.sign();
+		let output = json!({
+			"pcAddr": self.partnerchain_address,
+			"signature": signature,
+			"verificationKey": self.signing_key.vkey()
```

Comment:
Could you please make names more verbose and also update in the pallet in call to be exactly the same (verficationKey vs mcVkey), please?